src/libraries/System.Private.Xml/src/System/Xml/Core/XmlAsyncCheckWriter.cs

```diff
@@ -330,17 +330,6 @@ public override void WriteNode(XPathNavigator navigator, bool defattr)
             _coreWriter.WriteNode(navigator, defattr);
         }
 
-        protected override void Dispose(bool disposing)
```

Comment:
Why is this being deleted?

Reply:
This wrapper is for async only streams, I suppose do not need to override sync dispose

Reply:
Are you not allowed to use Dispose on such a stream? Are you saying this was actually dead code that wasn't reachable?

Reply:
> Are you not allowed to use Dispose on such a stream? Are you saying this was actually dead code that wasn't reachable?

No it was used, i thought we don't need it anymore as we are having `DisposeAsync` now after reading `Are you not allowed to use Dispose on such a stream?` just realized it still can use Dispose for such stream, will revert, thanks